images/ceph/toolbox.sh

```diff
@@ -64,7 +64,7 @@ watch_endpoints() {
 }
 
 # create the keyring file
-cat <<EOF > ${KEYRING_FILE}
+sudo cat <<EOF > ${KEYRING_FILE}
```

Comment:
@travisn I have changed here (not tested yet!) after removing `privileged` because it kept giving  an error as `/usr/local/bin/toolbox.sh: line 67: /etc/ceph/keyring: Permission denied`

Reply:
Ok, let's find a way to init the keyring and ceph.conf in `/etc/ceph` without running privileged. We might need to do this by mounting a configmap to this location if the `cp` fails